README.md

```diff
@@ -103,17 +120,17 @@ those dependencies.
 
 [venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
 
-```console
+```bash
 $ pip install -r requirements-dev.txt -r requirements.txt
 ```
 
 Then, point your environment to this directory:
 
-```console
+```bash
 $ python setup.py develop
 ```
 
-Now, you can edit the files and they will be automatically be seen by your
+Now, you can edit the files and they will be automatically seen by your
```

Comment:
Skip the Now. Simply: > Edit the files which will be automatically detected by your environment.

Reply:
better use:
...they will be seen automatically by your...